src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-server.h

```diff
@@ -0,0 +1,103 @@
+/**
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "laundry-washer-controls-delegate.h"
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/util/af.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace LaundryWasherControls {
+
+/**
+ * @brief LaundryWasherControls Server Plugin class
+ */
+class LaundryWasherControlsServer : public AttributeAccessInterface
+{
+public:
+    LaundryWasherControlsServer() : AttributeAccessInterface(Optional<EndpointId>::Missing(), LaundryWasherControls::Id) {}
+    static LaundryWasherControlsServer & Instance();
+
+    static constexpr uint8_t kMaxSpinSpeedLength = 64;
+
+    /**
+     * Set the default delegate of laundry washer server at endpoint x
+     * @param endpoint ID of the endpoint
+     * @param delegate The default delegate at the endpoint
+     * @return CHIP_ERROR CHIP_NO_ERROR on success, or corresponding error code.
+     */
+    static void SetDefaultDelegate(EndpointId endpoint, Delegate * delegate);
+
+    /**
+     * Init the laundry washer server.
+     * @param void
+     * @return CHIP_ERROR CHIP_NO_ERROR on success, or corresponding error code.
+     */
+    //    CHIP_ERROR Init();
+
+    /**
+     * @brief Set the attribute newSpinSpeedCurrent
+     *
+     * @param endpointId ID of the endpoint
+     * @param newSpinSpeedCurrent attribute SpinSpeedCurrent
+     * @return true on success, false on failure
+     */
+    EmberAfStatus SetSpinSpeedCurrent(EndpointId endpointId, DataModel::Nullable<uint8_t> newSpinSpeedCurrent);
+
+    /**
+     * @brief Get the attribute newSpinSpeedCurrent
+     *
+     * @param endpointId ID of the endpoint
+     * @param SpinSpeedCurrent attribute SpinSpeedCurrent
+     * @return true on success, false on failure
```

Comment:
commenting post merge, so I do not block the PR: these return EmberAfStatus, so probably not true on success/false on failure. Also the comment seems redundant - when something returns CHIP_ERROR or EmberAfStatus the return is reasonably clear.

Overall we can save vertical space a lot (it is obvious `GetSpinSpeedCurrent` gets the speed speed current). I guess it is unclear what `newSpinSpeedCurrent` is (what is `new`) but the comment does not help me much ... it seems everything is called newSpinSpeedCurrent.

Reply:
Thank you for the suggestions. I will change these in the next PR.

Reply:
I add the change in https://github.com/project-chip/connectedhomeip/pull/28264
Please review. Thank you